.gitignore

```diff
@@ -1,4 +0,0 @@
-_site/
-.sass-cache/
-.jekyll-metadata
-*~
```

Comment:
It is not fixed. The radioactive decay branch is fine, but this branch is not fine. We can talk about it on the call maybe if it's not clear.

This PR contains the commits in the radioactive decay branch. It also contains the commits that delete the radioactive decay files (as well as the style guide?). It also includes the commits that add the atomic structure lesson. Accordingly, when it is pulled, it will add the radioactive decay 
 files In so doing, it will merge all of the commits that are currently in the radioactive decay branch. So, that PR will be automatically closed. It will then delete the radioactive decay lesson. It will then add the atomic structure lesson. Does that make sense?